### ISSUE TITLE
apk for f-droid.org

### ISSUE THREAD

**Issue by hacksenwerk** [SecUSo/Aktivpause#13]
Hi. 
Can you please add a f-droid repo?

**Comment by jahway603** [comment]
I agree with this ghost user, but until then we can use [Obtainium]https://github.com/ImranR98/Obtainium].
